frontend/explorer/public/index.html

```diff
@@ -26,7 +26,7 @@
 	</div>
 
 	<div class='banner'>
-		<h1 class='banner-head'>ThreeFold Chain Explorer</h1>
```

Comment:
we never said to remove the class. The class should be kept for css purposes, you'll need to give it a unique ID name such as 'page-title'.